Source/WebKit/UIProcess/mac/WKFullScreenWindowController.h

```diff
@@ -71,11 +71,11 @@ typedef enum FullScreenState : NSInteger FullScreenState;
 
 @property (readonly) NSRect initialFrame;
 @property (readonly) NSRect finalFrame;
-@property (assign) NSArray *savedConstraints;
```

Comment:
I think we wanna keep the property as a raw pointer value and change its corresponding ivar to be `RetainPtr`.

Reply:
I removed that part.